mystery-deckbuilder/Assets/Scripts/Encounter/EncounterScript.cs

```diff
@@ -7,31 +8,97 @@
  * script for the encounter gameobject
  * to use:
  *      encounter = Instantiate(encounter);
-        encounter.GetComponent<EncounterScript>().StartEncounter(int complianceThreshold, int startingCompliance, int startingPatience)
+        encounter.GetComponent<EncounterScript>().StartEncounter()
  */
 public class EncounterScript : MonoBehaviour
 {
     [SerializeField] private int _threshold;
     [SerializeField] private int _compliance;
     [SerializeField] private int _patience;
+
+    [SerializeField] private string _weakness; //using strings because that is how element is stored in the card class
+    [SerializeField] private string _resistance; // !!!CARD CLASS AND NPC CLASS STRINGS MUST MATCH!!! 
+
+    [SerializeField] private int _handSize = 5;
+    private int playCount = 0;
+
     private int maxPatience = 10; //Not sure if this will ever change, but the option is there
 
-    private ArrayList deck; //TODO: this script will need a reference to the deck in whatever form that may take
+    private List<int> deck = new List<int>(); //reference to a list of card id numbers that make up the deck 
+    private List<int> handBackend = new List<int>(); //purgatory state between being in the deck and being discarded.
+    private List<int> discard = new List<int>(); //list of cards that have been played and removed from the deck
+
+    private List<ConversationCard> hand = new List<ConversationCard>(); //something weird is going on with all the "hands" I'll sort it out later (probably not)
```

Comment:
I'll add creating an interface base class for Cards to the Trello and try to get Michael on it immediately post-alpha